framework/src/problems/FEProblemBase.C

```diff
@@ -255,6 +255,11 @@ FEProblemBase::validParams()
       "and Jacobians (Kernels, BCs, etc.) by setting tags on them. The outer index is for which "
       "nonlinear system the extra tag vectors should be added for");
 
+  params.addParam<std::vector<std::vector<TagName>>>(
+      "not_zeroed_tag_vectors",
+      {},
+      "Extra vector tags which the sytem will not zero when other vector tags are zeroed");
```

Comment:
I would mention what the outer index is for (nonlinear system)

Reply:
Added the same additional description as for extra_tag_vectors.